Add KIC 1.3.0 (#2895)
Kong/docs.konghq.com: app/kubernetes-ingress-controller/1.3.x/references/annotations.md

Enables or disables response buffering on the Kong route associated with this
Ingress.

Sample usage:

```yaml
konghq.com/response-buffering: "false"
```

### konghq.com/host-aliases

> Available since controller 1.3

Set addtional hosts for routes created from rules on this Ingress

Sample usage:

```yaml
konghq.com/host-aliases: "example.com,example.net"
```

Note that this annotation applies to all rules equally. An Ingress like this:

```
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: example-ingress
  annotations:
    konghq.com/host-aliases: "example.com,example.net"
spec:
  rules:
  - host: "foo.example"
    http:
      paths:
      - pathType: Prefix
        path: "/bar"
        backend:
          service:
            name: service1
            port:
              number: 80
  - host: "bar.example"
    http:
      paths:
      - pathType: Prefix
        path: "/bar"
        backend:
          service:
            name: service2
            port:
              number: 80
```

Will result in two routes:

```
{"hosts":["foo.example", "example.com", "example.net"], "paths":["/foo"]}
{"hosts":["bar.example", "example.com", "example.net"], "paths":["/bar"]}

> Take care not to accidentally create unintentionally overlapping routes by not
> including the same path in multiple rules.

Reviewer: ```suggestion

{:.important}
> To avoid creating overlapping routes, don't reuse the same path in multiple rules.
```
I hope I understood what you were going for correctly, the number of negatives was a bit confusing. Got rid of both "accidentally" and "unintentional", as they're adding complexity - intentionality is up to the user to decide, and we should just warn them about the possibility and how to avoid it.
